README.md

```diff
@@ -1,31 +1,45 @@
-comtypes
-========
+# comtypes
 
-**comtypes** is a lightweight Python COM package, based on the ctypes_
-FFI library, in less than 10000 lines of code (not counting the
-tests).
+[![PyPI version](https://badge.fury.io/py/comtypes.svg)](https://pypi.org/project/comtypes/) [![PyPI - Python Version](https://img.shields.io/pypi/pyversions/comtypes)](https://pypi.org/project/comtypes/) [![PyPI - License](https://img.shields.io/pypi/l/comtypes)](https://pypi.org/project/comtypes/) [![PyPI - Downloads](https://img.shields.io/pypi/dd/comtypes)](https://pypi.org/project/comtypes/)
+[![GitHub Repo stars](https://img.shields.io/github/stars/enthought/comtypes?style=social)](https://github.com/enthought/comtypes/stargazers) [![GitHub forks](https://img.shields.io/github/forks/enthought/comtypes?style=social)](https://github.com/enthought/comtypes/network/members)
 
-**comtypes** allows to define, call, and implement custom and
-dispatch-based COM interfaces in pure Python.  It works on Windows and
-64-bit Windows. Windows CE support was dropped since comtypes==1.1.8.
+## About
 
-Documentation:
+`comtypes` is a lightweight `Python` COM package, based on the [`ctypes`](https://docs.python.org/library/ctypes.html) FFI library.
 
-    https://pythonhosted.org/comtypes
+`comtypes` allows to define, call, and implement custom and dispatch-based COM interfaces in pure `Python`.
 
-    Contribute using the `source repository and issue tracker
-    <https://github.com/enthought/comtypes/>`_ on GitHub.
+This package works on only Windows.
+- [`comtypes==1.1.7`](https://pypi.org/project/comtypes/1.1.7/) is the last version of supporting Windows CE.
 
-Mailing list:
+## Where to get it
 
-    http://gmane.org/info.php?group=gmane.comp.python.comtypes.user
+The source code is currently hosted on GitHub at:
+https://github.com/enthought/comtypes
 
-    https://lists.sourceforge.net/lists/listinfo/comtypes-users/
+Binary installers for the latest released version are available at the [Python Package Index (PyPI)](https://pypi.org/project/comtypes).
```

Comment:
What does it mean "binary" for source only package? Do you mean wheel packages? I'd suggest to leave just `An installer for the latest released version is ...`

Reply:
I have seen packages that use this wording (such as `pandas`) and I thought it may be customary in Python communities.

Your suggestion is more straightforward, so I will change it to that.